### PR TITLE
Content update and merge branches

### PR DESCRIPTION
**Pull request by nkoeries**

### CODE REVIEW

File: public-health/text/04-14.md [hunk 1]
```diff
@@ -84,7 +84,11 @@ No. By itself the accreditation process does not improve quality of care but it
 
 Targets can be set by a variety of managers, leaders or team members. For example:
 
+<<<<<<< HEAD:public-health/text/14.md
+* Targets for activities (such as the number of patients seen) and results (such as the number of wound infections) are usually part of the performance agreement that district managers have with provincial departments of health. This is part of health service governance.
+=======
 * Targets for activities (such as the number of patients seen) and results (such as the number of wound infections) are usually part of the performance agreement that district managers have with provincial departments of health. This is part of health service governance. 
+>>>>>>> BettercareHealth/master:public-health/text/04-14.md
 * The Millennium Development Goals were targets for poverty reduction and health improvement that were agreed between member governments of the United Nations. They were replaced in 2015 by the Sustainable Development Goals 2030. Goal 3 is Good Health and Well-being.
```
Comment (arthurattwell) [inline public-health/text/04-14.md]: @nkoeries A git conflict has snuck in here. Can you fix this and commit the fixed markdown file, please?

Comment (nkoeries) [reply public-health/text/04-14.md]: @arthurattwell fixed. Sorry about that.

Comment (arthurattwell) [reply public-health/text/04-14.md]: @nkoeries Thanks. The conflict is still here -- have you pushed/synced your commits?

Comment (arthurattwell) [reply public-health/text/04-14.md]: @nkoeries Cool, fixes showing here now. Thanks.

**Pull request merged**